src/Illuminate/Encryption/Encrypter.php

```diff
@@ -33,16 +34,23 @@ class Encrypter implements EncrypterContract
      * Create a new encrypter instance.
      *
      * @param  string  $key
-     * @param  string|null  $cipher
+     * @param  string  $cipher
      * @return void
      */
-    public function __construct($key, $cipher = null)
+    public function __construct($key, $cipher = 'AES-128-CBC')
     {
-        $this->key = (string) $key;
+        $len = mb_strlen($key = (string) $key, '8bit');
 
-        if ($cipher) {
+        if ($len === 16 || $len === 32) {
```

Comment:
@ircmaxell Does that mean previous versions of laravel have been doing this wrong? In 5.0 we have a 32 byte key and AES-128 out of the box.